draft-ietf-rats-reference-interaction-models.md

```diff
@@ -311,7 +303,7 @@ The following subsections introduce and illustrate the interaction models:
 3. Streaming Remote Attestation
 
 Each section starts with a sequence diagram illustrating the interactions between Attester and Verifier.
-While the presented interaction models focus on the conveyance of Evidence, the intention of this document is in support of future work that applies the presented models to the conveyance of other Conceptual Messages, namely Attestation Results, Endorsements, Reference Values, or Appraisal Policies.
+While the presented interaction models focus on the conveyance of Evidence, the intention of this document is in support of any type of Conceptual Messages conveyance that is based on the presented models.
```

Comment:
How about:

```suggestion
While the presented interaction models are illustrated by using the conveyance of Evidence as an example, the models presented can be adapted to the conveyance of other Conceptual Messages.
```

Shorter, more crisp, and maybe less offending?

Reply:
Sorry to pester but it's not about form.  It's about the content.

I disagree about making the claim that _"[t]he models presented can be adapted to the conveyance of other Conceptual Messages."_ without describing that in practice.  

As an example that would need to be articulated clearly in order to claim that the model fits more widely: what does Handle means in the context of conveyance of Reference Values?

Reply:
Handle would mean the exact same thing. It would be a Claim value used in the interaction model to demonstrate recent response and to prevent reply attack of RVs that might change the next session, so the requester of RV is able to tell that there has been a best effort to demonstrate freshness (an assertion in a message is reflecting reality) based on recentness (use of nonce).

Reply:
Would you be able to craft an appropriate example based on my illustration? I think it is important to get this right.

Reply:
> Handle would mean the exact same thing. It would be a Claim value used in the interaction model to demonstrate recent response and to prevent reply attack of RVs that might change the next session, so the requester of RV is able to tell that there has been a best effort to demonstrate freshness (an assertion in a message is reflecting reality) based on recentness (use of Handle).

I don't expect freshness of RVs to be provided by handles - see CoRIM, which relies on synchronised clocks between RV provider and Verifier.